astro/src/lib/session-store.ts

```diff
@@ -0,0 +1,137 @@
+import { v4 as uuidv4 } from 'uuid';
+
+export interface PurchaseSession {
+  sessionId: string;
+  purchaseCode: string;
+  email: string;
+  theme?: string;
+  kitType?: string;
+  organization?: string;
+  createdAt: number;
+  expiresAt: number;
+  browserFingerprint: string;
+  ipAddress?: string;
+}
+
+const SESSION_DURATION_MS = 30 * 60 * 1000;
```

Comment:
comment might be helpful here, also can we just input the values or why the multiplication?